src/geode/mesh/helpers/convert_solid_mesh.cpp

```diff
@@ -253,19 +414,29 @@ namespace geode
             }
             else if( vertices.size() == 8 )
             {
+                const auto ordered_vertices =
+                    order_hexahedron_vertices( p, solid );
                 builder->create_hexahedron(
-                    { vertices[0], vertices[1], vertices[2], vertices[3],
-                        vertices[4], vertices[5], vertices[6], vertices[7] } );
+                    { ordered_vertices[0], ordered_vertices[1],
+                        ordered_vertices[2], ordered_vertices[3],
+                        ordered_vertices[4], ordered_vertices[5],
+                        ordered_vertices[6], ordered_vertices[7] } );
```

Comment:
I think you can give the ordered_vertices array directly, since it should have the right type